app/assets/javascripts/pages/agent-edit-page.js.coffee

```diff
@@ -108,10 +108,12 @@ class @AgentEditPage
     $(".control-link-region").show()
 
   hideEventCreation: ->
-    $(".event-related-region").hide()
```

Comment:
@TildeWill That is equivalent to the setup in this PR, right? Even with the `transaction` setting `database_clean` is a lot slower then the build in `use_transactional_fixtures`:

```
RSPEC_TASK=spec:nofeatures rake spec:nofeatures # No database cleaner
Finished in 41.1 seconds (files took 5.05 seconds to load)

rake spec:nofeatures # database_cleaner in transaction mode
Finished in 1 minute 44.15 seconds (files took 5.14 seconds to load)
```

I initially thought splitting up the unit and feature specs would be trickier, but think it works rather well. You can still use `rake` locally to run everything and speed it up with `RSPEC_TASK=spec:nofeatures rake spec:nofeatures` when needed.